deeplinkdispatch-processor/build.gradle

```diff
@@ -13,7 +13,8 @@ dependencies {
     testCompile 'junit:junit:4.12'
     testCompile 'org.assertj:assertj-core:1.7.0'
     testCompile 'com.google.android:android:4.1.1.4'
-    testCompile 'com.google.android:support-v4:r7'
+    //testCompile 'com.google.android:support-v4:r7'
+    testCompile fileTree(dir: 'libs', include: ['android-support-v4.jar'])
```

Comment:
if i revert this then DeepLinkProcessor tests fail to compile as v4:r7 doesn't contain `TaskStackBuillder.editIntentAt(int)` method  but  ['getIntent(int index)'](https://developer.android.com/reference/android/support/v4/app/TaskStackBuilder.html#getIntent(int))  which is deprecated now